tests/test_parsl_flux_mpi_hello.py

```diff
@@ -123,6 +123,12 @@ def test_run_mpi_hello(load_config):
                           ).result()
     assert hello == 0
 
+    with open('parsl_flux_mpi_hello_run.out', 'r') as f:
+        print("file", f.read())
+        for line in f:
+            assert re.match(r'ƒ\S+: Hello world from host -\S+, rank \d+ out of 6.', f.read())
```

Comment:
Are you sure that `ƒ\S+:` is part of the pattern?  When I look at the output after running it, I don't see that at the beginning of the lines.  Do you?  I only see:
```
(base) christopher.w.harrop@harrop-lt tests % cat parsl_flux_mpi_hello_run.out
Hello world from host slurmnode1, rank 0 out of 6
Hello world from host slurmnode1, rank 1 out of 6
Hello world from host slurmnode2, rank 2 out of 6
Hello world from host slurmnode2, rank 3 out of 6
Hello world from host slurmnode3, rank 4 out of 6
Hello world from host slurmnode3, rank 5 out of 6
(base) christopher.w.harrop@harrop-lt tests % 
```

Reply:
This is what I have for my output file:
```
(chiltepin) hfe12:/scratch1/BMC/zrtrr/Naureen.Bharwani/SENA/ExascaleWorkflowSandbox/tests $ cat parsl_flux_mpi_hello_run.out 
Hello world from host h23c01, rank 0 out of 6
Hello world from host h23c01, rank 1 out of 6
Hello world from host h23c11, rank 2 out of 6
Hello world from host h23c11, rank 3 out of 6
Hello world from host h23c12, rank 4 out of 6
Hello world from host h23c12, rank 5 out of 6
(chiltepin) hfe12:/scratch1/BMC/zrtrr/Naureen.Bharwani/SENA/ExascaleWorkflowSandbox/tests $
```

I'm debugging currently and the big cause of my issues (using ChatGPT as a resource) is I had a trailing `.` in my match statement that is not in the output file. Still working on this, but hopefully that will help move things along.